build/patternfly-cli/pf.js

```diff
@@ -66,13 +66,21 @@ program
       nameCapitalized: inflection.capitalize(name),
       bemName: `${bemName}`,
       bemModifierBlock: `{{#if ${bemEntity}--modifier}} {{${bemEntity}--modifier}}{{/if}}`,
+      bemItemModifierBlock: `{{#if ${bemEntity}-item--modifier}} {{${bemEntity}-item--modifier}}{{/if}}`,
       moduleName,
       partialBlock: '{{> @partial-block}}',
-      attributeBlock: `{{#if ${bemEntity}--attribute}}
+      attributeBlock: `
+  {{#if ${bemEntity}--attribute}}
```

Comment:
This is creating an empty line above the attribute block in the main component hbs partial